### PR TITLE
fix: fully implement rule 219

### PR DESCRIPTION
**Pull request by dweber019**

### CODE REVIEW

**Review by leubedane — CHANGES_REQUESTED** [review]

File: zalando.yml [hunk 1]
```diff
@@ -426,15 +426,22 @@ rules:
...
     given: $.info
     then:
-      field: x-audience
-      function: truthy
+      - field: x-audience
+        function: truthy
+      - field: x-audience
+        function: pattern
+        functionOptions:
+          match: ^(component-internal|business-unit-internal|company-internal|external-partner|external-public)$
```
[the remark this answers is not in the document]
Comment (dweber019) [reply zalando.yml:444]: agree will change it

[the remark this answers is not in the document]
Comment (leubedane) [reply zalando.yml:444]: We will also need to add an entry to our documentation, i guess

[the remark this answers is not in the document]
Comment (leubedane) [reply zalando.yml:444]: Done

File: baloise.yml [hunk 2]
```diff
@@ -108,4 +108,24 @@ rules:
...
+    documentationUrl: https://github.com/baloise-incubator/spectral-ruleset/blob/main/doc/rules/property-names-must-be-ascii-camel-case.md
+    severity: error
+    given: $.info
+    then:
+      - field: x-audience
+        function: truthy
+      - field: x-audience
+        function: pattern
+        functionOptions:
+          match: ^(component-internal|business-unit-internal|company-internal|external-partner|external-public)$
```
[the remark this answers is not in the document]
Comment (leubedane) [reply baloise.yml:131]: You are right, the change is missing, I will fix it :-)

[the remark this answers is not in the document]
Comment (leubedane) [reply baloise.yml]: Now it should be right. @dweber019  and @joachimprinzbach

**Review by leubedane — APPROVED** [review]

**Pull request merged**